packages/container/lib/container.js

```diff
@@ -1,6 +1,6 @@
 import { dictionary, symbol, setOwner, OWNER } from 'ember-utils';
 import { ENV } from 'ember-environment';
-import { assert, deprecate, runInDebug } from 'ember-metal';
+import { assert, deprecate, runInDebug, NAME_KEY } from 'ember-metal';
```

Comment:
i made the change, but something after moving that stuff around has broken things ... need to figure out what